src/main/keep-container/keep-content/keep-search/keep-search.component.html

```diff
@@ -0,0 +1,10 @@
+<app-keep-common-note [isArchiveNotePresent]="false" [isSearch]="true"></app-keep-common-note>
+
+<div *ngIf="isNotesSimpleAndArchive() ">
+    <div class="text-archive">Archive</div>
+</div>
+<app-keep-common-note [isArchiveNotePresent]="true" [isSearch]="true"></app-keep-common-note>
+
+<div *ngIf=" (!(hasNotes()) ||isMatchedNotes()) && searchValue!='' " class="box">
```

Comment:
we can simplify above condition as we discussed:
!isMatchedNotes && searchValue

Reply:
yes done